src/services/notifications/templates/promotional-email.tsx

```
@@ -19,30 +21,45 @@ export function PromotionalEmail({
-          padding: "12px 24px",
-          backgroundColor: "#14a800",
-          color: "white",
-          textDecoration: "none",
-          borderRadius: "4px",
-          fontWeight: "bold",
-        }}
-      >
-        {ctaText}
-      </a>
+        {/* Safely inject campaign content (already sanitized or trusted HTML) */}
+        <div dangerouslySetInnerHTML={{ __html: content }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)